### PR TITLE
Add notebook and env for s3 minio

### PR DESCRIPTION
**Pull request by pwalczysko**
WiP

cc @jburel - it builds fine for me with mamba on the old mac - is this what we want ?

### CODE REVIEW

File: Day_5/environment.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+channels:
+  - bioconda
+  - conda-forge
+  - defaults
+  - ome
+  - joshmoore
```
Comment (jburel) [inline Day_5/environment.yml:6]: do we still need that channel?

Comment (joshmoore) [reply Day_5/environment.yml:6]: If you want the build of go-mc, then yes. We can migrate it to the org though if you'd like.

Comment (jburel) [reply Day_5/environment.yml:6]: I assumed that got moved to a more "official" channel. If we start using it, and I believe we might I think we should move it out of your channel at some stage. Is it something that could be moved to ``conda-forge`` (After some work!) @joshmoore and be useful for community? If this is the case, I will keep it for now under your channel and move to ``conda-forge`` when ready

**Comment by jburel** [comment]
@pwalczysko it is marked as WIP but i will need the structure for Day_5

**Comment by jburel** [comment]
Discussed with @pwalczysko. Merging

File: Day_5/environment.yml (hunk 1 above)
Comment (joshmoore) [reply Day_5/environment.yml:6]: I've begun setting up https://github.com/ome/go-mc-feedstock

(Old repository will stay in place to not break existing notebooks)

**Pull request merged**